proto/o5/dante/v1/dead_message.proto

```diff
@@ -54,3 +71,55 @@ message Any {
   google.protobuf.Any proto = 1;
   string json = 2;
 }
+
+message DeadMessageEvent {
+  Metadata metadata = 1 [
+    (buf.validate.field).required = true,
+    (psm.v1.event_field).metadata = true
+  ];
+
+  string message_id = 2 [
+    (buf.validate.field).string.uuid = true,
+    (psm.v1.event_field).state_key = true
+  ];
+
+  DeadMessageEventType event = 3 [
+    (psm.v1.event_field).event_type = true
+  ];
+}
+
+message DeadMessageEventType {
+  oneof type {
+    Created  created  = 1;
+    Updated  updated  = 2;
+    Replayed replayed = 3;
+    Rejected rejected = 4;
```

Comment:
If I'm following the state diagram, shouldn't this state be "shelved?"

Reply:
Yup it should. I knew there was gonna be something I missed in all of that.

Reply:
fixed the state diagram to match this, and the current convo.